source/features/prevent-link-loss.tsx

```diff
@@ -7,27 +7,50 @@ import * as pageDetect from 'github-url-detection';
 import * as textFieldEdit from 'text-field-edit';
 
 import features from '.';
-import {prCommitUrlRegex, preventPrCommitLinkLoss, prCompareUrlRegex, preventPrCompareLinkLoss} from '../github-helpers';
+import {
+	prCommitUrlRegex,
+	preventPrCommitLinkLoss,
+	prCompareUrlRegex,
+	preventPrCompareLinkLoss,
+	discussionUrlRegex,
+	preventDiscussionLinkLoss
+} from '../github-helpers/prevent-link-loss';
 
 function handleButtonClick({delegateTarget: fixButton}: delegate.Event<MouseEvent, HTMLButtonElement>): void {
 	const field = fixButton.form!.querySelector('textarea')!;
 	textFieldEdit.replace(field, prCommitUrlRegex, preventPrCommitLinkLoss);
 	textFieldEdit.replace(field, prCompareUrlRegex, preventPrCompareLinkLoss);
+	textFieldEdit.replace(field, discussionUrlRegex, preventDiscussionLinkLoss);
 	fixButton.parentElement!.remove();
 }
 
+function createRghIssueLink(issueNumber: number): Element {
+	const issueLink = `https://github.com/sindresorhus/refined-github/issues/${issueNumber}`;
+	return (
+		<a target="_blank" rel="noopener noreferrer" data-hovercard-type="issue" data-hovercard-url={issueLink + '/hovercard'} href={issueLink}>
+			#{issueNumber}
+		</a>
+	);
+}
```

Comment:
Can you extract this function to its own helper? It will replace https://github.com/sindresorhus/refined-github/pull/4394/files#diff-cf4a8817af449037372dff756e54ce5c3c36a4c66d0097231c93b030e7664bf5

This PR will be merged before that one